ios/Loki Network/Loki Network.entitlements

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>com.apple.developer.networking.networkextension</key>
+	<array>
+		<string>packet-tunnel-provider</string>
+	</array>
+	<key>com.apple.developer.networking.vpn.api</key>
+	<array>
+		<string>allow-vpn</string>
+	</array>
+	<key>com.apple.security.application-groups</key>
+	<array>
+		<string>group.com.niels-andriesse.loki-network</string>
```

Comment:
This group should use loki-controlled namespace, if this in included in the repo at all.

Team Apple Developer accounts have been discussed with Simon before as well, and agreed to, if ease of signing and such is a concern.

Reply:
See comment above